pkg/jaeger/store/store.go

```diff
@@ -86,8 +88,10 @@ func (p *Store) GetServices(ctx context.Context) ([]string, error) {
 	code := "5xx"
 	start := time.Now()
 	defer func() {
-		metrics.Query.With(prometheus.Labels{"type": "trace", "handler": "Get_Services", "code": code}).Inc()
-		metrics.QueryDuration.With(prometheus.Labels{"type": "trace", "handler": "Get_Services", "code": code}).Observe(time.Since(start).Seconds())
+		labels := prometheus.Labels{"type": "trace", "handler": "Get_Services", "code": code, "err": ""}
+		metrics.Query.With(labels).Inc()
+		delete(labels, "err")
+		metrics.QueryDuration.With(labels).Observe(time.Since(start).Seconds())
```

Comment:
May be using `WithLabelValues`  would make less verbose?
```suggestion
		metrics.Query.WithLabelValues("trace", "Get_Services", code, "").Inc()
		metrics.QueryDuration.WithLabelValues("trace", "Get_Services", code).Observe(time.Since(start).Seconds())
```
WDYT?

Reply:
That's definitely better for verbosity, however then we need to strictly follow the order of labels and make sure it does not change over time. So I'd rather keep it like this, mostly to be on the safer side (also hard to catch these problems). Btw I am not strongly opinionated here so let me know if you feel strongly about this.